native-app-flask-example/README.md

```diff
@@ -13,24 +13,17 @@ To build and run the example application you will need:
 
 ## Set up the development environment
 
-First, you will need to create an EDM environment named `dev_env` and install some dependencies.
+First, you will need to create an EDM environment and install some dependencies.
+To do this, use the "bootstrap.py" script:
 
 ```commandline
-edm install -e dev_env --version 3.8 -y click \
-    requests \
-    opencv_python \
-    enthought_edge \
-    "flask>2" && \
-edm run -e dev_env -- python -m pip install "jupyterhub==2.2.2" \
-    dockerspawner \
-    "configurable-http-proxy" \
-    Flask-Session
-```   
+python bootstrap.py
```

Comment:
Should this also be applied to the dashboard app?

Reply:
Yes, it was already applied in #17.